buildspec-pr.yaml

```diff
@@ -16,6 +16,9 @@ phases:
       # Install yarn if it wasn't already present in the image
       - yarn --version || npm -g install yarn
 
+      # Install pnpm if it wasn't already present in the image
+      - pnpm --version || npm -g install pnpm
```

Comment:
We shouldn't need this.

Reply:
I get `bash: pnpm: command not found` if I remove it, although I'm not sure if it's needed for just one of the files or both

![image](https://user-images.githubusercontent.com/60179183/208692643-7698d903-2002-4a19-b8c7-a06f0c7ccff8.png)